tests/mgr/test_dashboard_plugin_with_ceph-mgr.py

```diff
@@ -0,0 +1,34 @@
+from cli.cephadm.cephadm import CephAdm
```

Comment:
change name to `test_dashboard_plugin_with_ceph_mgr.py`

Reply:
Removed